<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>924</width>
    <height>758</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="horizontalLayoutWidget_5">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>10</y>
      <width>631</width>
      <height>101</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="handLayout2">
     <item>
      <widget class="QListWidget" name="enemyHandlistWidget"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_6">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>110</y>
      <width>631</width>
      <height>101</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="spellLayout2">
     <item>
      <widget class="QTextEdit" name="textEdit_8"/>
     </item>
     <item>
      <widget class="QTextEdit" name="textEdit_10"/>
     </item>
     <item>
      <widget class="QTextEdit" name="textEdit_9"/>
     </item>
     <item>
      <widget class="QTextEdit" name="textEdit_7"/>
     </item>
     <item>
      <widget class="QTextEdit" name="textEdit_6"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_7">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>210</y>
      <width>631</width>
      <height>101</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="monsterLayout2">
     <item>
      <widget class="QListWidget" name="enemymonsterWidget1"/>
     </item>
     <item>
      <widget class="QListWidget" name="enemymonsterWidget2"/>
     </item>
     <item>
      <widget class="QListWidget" name="enemymonsterWidget3"/>
     </item>
     <item>
      <widget class="QListWidget" name="enemymonsterWidget4"/>
     </item>
     <item>
      <widget class="QListWidget" name="enemymonsterWidget5"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_8">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>310</y>
      <width>631</width>
      <height>101</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="rinkLayout"/>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_9">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>410</y>
      <width>631</width>
      <height>101</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="monsterLayout1">
     <item>
      <widget class="QListWidget" name="monsterWidget1"/>
     </item>
     <item>
      <widget class="QListWidget" name="monsterWidget2"/>
     </item>
     <item>
      <widget class="QListWidget" name="monsterWidget3"/>
     </item>
     <item>
      <widget class="QListWidget" name="monsterWidget4"/>
     </item>
     <item>
      <widget class="QListWidget" name="monsterWidget5"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_10">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>510</y>
      <width>631</width>
      <height>101</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="spellLayout1">
     <item>
      <widget class="QTextEdit" name="textEdit_19"/>
     </item>
     <item>
      <widget class="QTextEdit" name="textEdit_20"/>
     </item>
     <item>
      <widget class="QTextEdit" name="textEdit_18"/>
     </item>
     <item>
      <widget class="QTextEdit" name="textEdit_17"/>
     </item>
     <item>
      <widget class="QTextEdit" name="textEdit_16"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_11">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>610</y>
      <width>631</width>
      <height>101</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="handLayout1">
     <item>
      <widget class="QListWidget" name="handlistWidget"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_12">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>510</y>
      <width>81</width>
      <height>101</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="fieldspellLayout1"/>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_13">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>500</y>
      <width>81</width>
      <height>101</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="graveyardLayout1"/>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_14">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>610</y>
      <width>81</width>
      <height>101</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="deckLayout1"/>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_15">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>400</y>
      <width>81</width>
      <height>101</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="banishzoneLayout1"/>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_16">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>220</y>
      <width>81</width>
      <height>101</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="banishzoneLayout2"/>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_18">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>120</y>
      <width>81</width>
      <height>101</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="graveyardLayout2"/>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_17">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>81</width>
      <height>101</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="deckLayout2"/>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_19">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>610</y>
      <width>81</width>
      <height>101</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="extradeckLayout1"/>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_20">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>110</y>
      <width>81</width>
      <height>101</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="fieldspellLayout2"/>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_21">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>10</y>
      <width>81</width>
      <height>101</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="extradeckLayout2"/>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>300</y>
      <width>121</width>
      <height>91</height>
     </rect>
    </property>
    <property name="text">
     <string>フェイズ進行</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="lifepointtextEdit2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>330</y>
      <width>104</width>
      <height>71</height>
     </rect>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
hr { height: 1px; border-width: 0; }
li.unchecked::marker { content: &quot;\2610&quot;; }
li.checked::marker { content: &quot;\2612&quot;; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Segoe UI'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;8000&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="lifepointtextEdit1">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>410</y>
      <width>104</width>
      <height>71</height>
     </rect>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
hr { height: 1px; border-width: 0; }
li.unchecked::marker { content: &quot;\2610&quot;; }
li.checked::marker { content: &quot;\2612&quot;; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Segoe UI'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;8000&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>924</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
